pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -252,109 +258,58 @@ func TestLocalTaskRef(t *testing.T) {
 				t.Error(diff.PrintWantGot(d))
 			}
 
-			// local cluster tasks have empty RefSource for now. This may be changed in future.
+			// local cluster tasks have empty source for now. This may be changed in future.
 			if refSource != nil {
-				t.Errorf("expected refSource is nil, but got %v", refSource)
+				t.Errorf("expected configsource is nil, but got %v", refSource)
 			}
 		})
 	}
 }
 
-// TestGetTaskFunc_Bundle tests the deprecated v1beta1 bundle syntax, this
-// can be removed when support for the bundle syntax is removed
-func TestGetTaskFunc_Bundle(t *testing.T) {
-	// Set up a fake registry to push an image to.
-	s := httptest.NewServer(registry.New())
-	defer s.Close()
-	u, err := url.Parse(s.URL)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	ctx := context.Background()
-
+func TestLocalTaskRef_Error(t *testing.T) {
```

Comment:
I'm a bit confused, is this a new test class? Does this test need to be added in this PR?

Reply:
same as https://github.com/tektoncd/pipeline/pull/6444#discussion_r1226529126, I opened https://github.com/tektoncd/pipeline/pull/6813 and removed from the changes in this PR